.ci/static-checks.sh

```diff
@@ -823,11 +823,28 @@ static_check_docs()
 	for doc in $docs
 	do
 		"$cmd" check "$doc" || { info "spell check failed for document $doc" && docs_failed=1; }
+
+		static_check_eof "$doc"
 	done
 
 	[ $docs_failed -eq 0 ] || die "spell check failed, See https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#spelling for more information."
 }
 
+static_check_eof()
+{
+	local file="$1"
+	local anchor="EOF"
+
```

Comment:
```suggestion
	[ -z "$file" ] && return 0
```
CI's error message This file may be empty